v0/destinations/fbpixel/transform.js

```diff
@@ -0,0 +1,431 @@
+      const productId = product.product_id;
+      if (productId) {
+        contentIds.push(productId);
+        contents.push({
+          id: productId,
+          quantity: message.properties.quantity
+        });
+      }
+    });
+  }
+
+  if (contentIds.length) {
+    contentType = "product";
+  } else {
+    contentIds.push(message.properties.category || "");
+    contents.push({
+      id: message.properties.category || "",
+      quantity: 1
+    });
+    contentType = "product_group";
+  }
+  return {
+    content_ids: contentIds,
+    content_type: getContentType(message, contentType, categoryToContent),
+    contents
+  };
+}
+
+function handleProduct(message, categoryToContent, valueFieldIdentifier) {
+  const useValue = valueFieldIdentifier === "properties.value";
+  const contentIds = [
+    message.properties.product_id ||
+      message.properties.id ||
+      message.properties.sku ||
+      ""
+  ];
+  const contentType = getContentType(message, "product", categoryToContent);
+  const contentName =
+    message.properties.product_name || message.properties.name || "";
+  const contentCategory = message.properties.category || "";
+  const { currency } = message.properties;
+  const value = useValue
+    ? formatRevenue(message.properties.value)
+    : formatRevenue(message.properties.price);
+  const contents = [
+    {
+      id:
+        message.properties.product_id ||
+        message.properties.id ||
+        message.properties.sku ||
+        "",
+      quantity: message.properties.quantity,
+      item_price: message.properties.price
+    }
+  ];
+  return {
+    content_ids: contentIds,
+    content_type: contentType,
+    content_name: contentName,
+    content_category: contentCategory,
+    currency,
+    value,
+    contents
+  };
+}
+
+function checkPiiProperties(
+  message,
+  custom_data,
+  blacklistPiiProperties,
+  whitelistPiiProperties,
+  isStandard,
+  eventCustomProperties
+) {
+  const defaultPiiProperties = [
+    "email",
+    "firstName",
+    "lastName",
+    "firstname",
+    "lastname",
+    "first_name",
+    "last_name",
+    "gender",
+    "city",
+    "country",
+    "phone",
+    "state",
+    "zip",
+    "birthday"
+  ];
+  blacklistPiiProperties = blacklistPiiProperties || [];
+  whitelistPiiProperties = whitelistPiiProperties || [];
+  const customPiiProperties = {};
+  for (let i = 0; i < blacklistPiiProperties.length; i += 1) {
+    const configuration = blacklistPiiProperties[i];
+    customPiiProperties[configuration.blacklistPiiProperties] =
+      configuration.blacklistPiiHash;
+  }
+  Object.keys(custom_data).forEach(function(property) {
+    const isPropertyPii = defaultPiiProperties.indexOf(property) >= 0;
+    let isProperyWhiteListed = false;
+    for (let i = 0; i < whitelistPiiProperties.length; i += 1) {
+      const configuration = whitelistPiiProperties[i];
+      const properties = configuration.whitelistPiiProperties;
+      if (properties === property) {
```

Comment:
change names (presently it look like a plural to singular comparison)

Reply:
done